docs/sdk/api/sdkMiddlewareAuth.md

```diff
@@ -60,12 +55,11 @@ Creates a [middleware](/sdk/Glossary.md#middleware) to handle authentication for
 
 #### Named arguments (options)
 
-1. `host` *(String)*: the host of the OAuth API service
-2. `projectKey` *(String)*: the key of the project to assign the default scope to
-   - The user field is an object containining `username` and `password`. [Sample below](#usage-example)
-3. `credentials` *(Object)*: the client credentials for authentication (`clientId`, `clientSecret`)
-4. `scopes` *(Array)*: a list of [scopes](http://dev.commercetools.com/http-api-authorization.html#scopes) to assign to the OAuth token. _No default scope is sent_
-
+1. `host` _(String)_: the host of the OAuth API service
+2. `projectKey` _(String)_: the key of the project to assign the default scope to
+3. `credentials` _(Object)_: the client credentials for authentication (`clientId`, `clientSecret`, `user`)
+  * The `user` field is an object containing `username` and `password`. [Sample below](#usage-example)
```

Comment:
(#usage-example) should be (#usage-example-1) here

Reply:
2489207 (via `git commit --amend`